examples/run/cpu/full_chain_algorithm.hpp

```diff
@@ -62,6 +63,8 @@ class full_chain_algorithm : public algorithm<track_state_container_types::host(
         traccc::host::combinatorial_kalman_filter_algorithm;
     /// Track fitting algorithm type
     using fitting_algorithm = traccc::host::kalman_fitting_algorithm;
+    // Uncomment (and comment line above) to use Triplet Fitting Algorithm
+    // using fitting_algorithm = traccc::host::triplet_fitting_algorithm;
```

Comment:
Let' not have this in the example like this. We might need to find a better way to select an algorithm later.
Adding tests would be enough for this PR.